models/vyos/zone_policy/vyos_zones.yaml

```diff
@@ -0,0 +1,66 @@
+DOCUMENTATION: |
+  module: vyos_firewall_zones
+  ## version_added: 2.10
+  short_description: FIREWALL zones resource module
+  description: This module manages firewall zones on VyOS devices
+  author:
+    - 
+  options:
+    config:
+      description: A dictionary of Firewall zone options.
+      type: list
+      elements: dict
+      suboptions:
+        zones:
+          description:
+            - The zones list.
+          type: list
+          elements: dict
+            suboptions:
+            zname: 
+              description:
+                - Specifies the name of the zone.
+              type: str
+              required: True
+              suboptions:
+                default-action:
+                  description:
+                    - Default-action for traffic coming into this zone.
+                    - drop (Drop silently (default))
+                    - reject (Drop and notify source)
+                  type: str
+                  choices: ['drop', 'reject']
+                description:
+                  description:
+                    - Zone description.
+                  type: str
+                from:
+                  description:
+                    - Zone from which to filter traffic.
+                  type: list
+                  elements: dict
+                  suboptions:
+                    name:
+                      description:
+                        - Zone from which to filter traffic.
+                      type: str
+                      required: True
+                    firewall:
+                      description:
+                        - Firewall options.
+                      type: dict
+                      choices: ['ipv6-name', 'name']
+                      suboptions:
+                        rule_set_name: 
+                          description:
+                            - firewall ruleset.
+                          type: str
+                          required: True
+                interface:
+                  description:
+                    - Interface associated with zone.
+                    type: str
+                local-zone:
+                  description:
+                    - Zone to be local-zone.
+                  type: str        
```

Comment:
add the state variable with supported operations options

Reply:
Added the state variable.